src/extension.ts

```diff
@@ -615,11 +616,31 @@ async function showCompilationDatabaseOptions(paths: VSCode.Uri[]) {
 
 function showMessageAndUpdateConfig(compilationDbPath: string) {
   VSCode.window.showInformationMessage(
-    `Analysis configured. Compilation database path is set to: ${compilationDbPath}`
+      `Analysis configured. Compilation database path is set to: ${compilationDbPath}`
   );
+  const [pathForSettings, workspaceFolder] = getPathWithWorkspaceVariableForPathIfInWorkspace(compilationDbPath);
+
+  if (workspaceFolder !== undefined) {
+    const config = VSCode.workspace.getConfiguration('sonarlint', workspaceFolder.uri);
+    return config.update('pathToCompileCommands', pathForSettings, VSCode.ConfigurationTarget.WorkspaceFolder);
```

Comment:
This is a kind of duplication of the constant. What about having something like:
```
const SONARLINT_CATEGORY='sonarlint';
const PATH_TO_COMPILE_COMMANDS = 'pathToCompileCommands';
const FULL_PATH_TO_COMPILE_COMMANDS=`${SONARLINT_CATEGORY}.${PATH_TO_COMPILE_COMMANDS}`
```